src/main/java/bio/terra/cloudfiletodatastore/ResourceFetcher.java

```diff
@@ -0,0 +1,18 @@
+package bio.terra.cloudfiletodatastore;
```

Comment:
We need to figure out whether using the slightly more unwieldy InputStream approach is worth the memory savings--something we'll probably need to load test.